packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -88,30 +89,7 @@ describe("MaintainOperatorValueService", () => {
         }
     }, 60 * 1000)
 
-    it("updates both sponsorships to stay over the threshold", async () => {
-        const penaltyFraction = parseEther("0.001")
-        const poolValue = STAKE_AMOUNT * 2
```

Comment:
What is the relation of const `poolValue = STAKE_AMOUNT * 2` and `SPONSOR_AMOUNT`? Seems that both have the same value of `200`.

Reply:
Pool value should actually be: `poolValue = STAKE_AMOUNT * 2 + 0` and zero would be the freeFunds (e.g. DATA balance in operator contract, not yet put to work). STAKE_AMOUNT is not dependent of SPONSOR_AMOUNT and vice-versa. They are different values and came form different sources (e.g. stake amount comes from user delegations and sponsor amount comes from user sponsorings).

Reply:
Would it make sense to have different values if not dependent?

Reply:
Doesn't hurt, for sure, but at the same time I'd say it doesn't really matter in this case. Changed it anyways to make it obvious that the sponsor amount does not need to be equal with the sum of the stakings.